Makefile

```diff
@@ -85,10 +88,6 @@ image-dev-full: image-dev build-drivers
 		--build-arg max_layer_depth=1 \
 		$(CURDIR)/kernel-modules/container
 
-.PHONY: integration-tests
-integration-tests:
```

Comment:
Why are we removing this target entirely? It might not be as convenient as some of the others, but if it works I don't see any harm in having it.

Reply:
I propose to remove it because the `tests` target doesn't exist anymore in the integration-tests subdir.

Reply:
I totally missed that, let's go ahead and get it outta here then!